opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/point.py

```diff
@@ -149,6 +158,21 @@ class ScopeMetrics:
     metrics: Sequence[Metric]
     schema_url: str
 
+    def to_json(self) -> str:
+        return dumps(
+            {
+                "scope": {
```

Comment:
Should we implement a `to_json` for `InstrumentationScope` as well?

Reply:
Not sure, really. This is the only place we need to "translate" `InstrumentationScope` to JSON.

Reply:
This is only because we did not follow OTLP data model for exporting `ReadableSpan`. Regardless, we might want to do the same thing that we did for metrics for logs, in which the exported data model contains `InstrumentationScope`, in which case `InstrumentationScope` would need a `to_json()`.

Reply:
Agree, we already have it implemented for `Resource`.

Reply:
All right, added `to_json` to `InstrumentationScope`.